versions.props

```diff
@@ -1,7 +1,7 @@
 com.fasterxml.jackson.core:jackson-databind = 2.13.3
 com.google.guava:guava = 31.1-jre
 com.palantir.safe-logging:* = 3.0.0
-junit:junit = 4.12
 org.assertj:assertj-core = 3.23.1
+org.junit.jupiter:* = 5.5.0
```

Comment:
Might as well jump to latest 5.9.0, 5.5.0 was released a bit over three years ago!

Reply:
Good call, I had copied this from an older project template. Will update when I can `write-locks` (note to self, look at nit bot for OSS repos)